src/main/java/com/example/ludogoriesoft/lukeriaerpapi/repository/ProductRepository.java

```diff
@@ -16,4 +16,8 @@ public interface ProductRepository extends JpaRepository<Product, Long> {
     Optional<Product> findByIdAndDeletedFalse(Long id);
 
     Optional<Product> findByPackageIdAndDeletedFalse(Package packageEntity);
+
+    @Query("SELECT p FROM Product p WHERE p.forSale=true")
```

Comment:
I don't think we need to use a query here, we can just write  List<Product> findByForSale(true); or List<Product> findByForSaleTrue();

Reply:
Does it really matter